servicex/resources/query_transformation_request.py

```diff
@@ -39,8 +40,13 @@ def get(self, request_id=None):
             return {'message': f'Authentication Failed: {str(auth_reject_message)}'}, 401
 
         if request_id:
-            return TransformRequest.to_json(
-                TransformRequest.return_request(request_id)
-            )
+            request_rec = TransformRequest.to_json(
+                TransformRequest.return_request(request_id))
+
+            if current_app.config['OBJECT_STORE_ENABLED']:
+                request_rec['minio-endpoint'] = current_app.config['MINIO_PUBLIC_URL']
+                request_rec['minio-access-key'] = current_app.config['MINIO_ACCESS_KEY']
+                request_rec['minio-secret-key'] = current_app.config['MINIO_SECRET_KEY']
+            return request_rec
         else:
```

Comment:
If the person requests a non-object store output (see the example Jason you have on the home page where Kafka is requested), will this send back this info anyway? Do we care if it does?

Reply:
Updated and also provided a test where Object store exists, but transform request goes to Kafka